can/interfaces/pcan/basic.py

```diff
@@ -967,7 +967,7 @@ def GetValue(self, Channel, Parameter):
             elif Parameter == PCAN_ATTACHED_CHANNELS:
                 res = self.GetValue(Channel, PCAN_ATTACHED_CHANNELS_COUNT)
                 if TPCANStatus(res[0]) != PCAN_ERROR_OK:
-                    return (TPCANStatus(res[0]),)
+                    return TPCANStatus(res[0]), ()
```

Comment:
```suggestion
                    return TPCANStatus(res[0]), None
```

Reply:
Since you expect to receive an Iterable, returning an empty tuple is better so you don't have to check if it is "None" before using it in a For loop.

Reply:
This is the error case. There is no valid result, so it is not used here:

https://github.com/hardbyte/python-can/blob/7dba4490c6c61385dc8cbe917e85492b4777b8f6/can/interfaces/pcan/pcan.py#L720